PROProtocol/PokemonExperience.cs

```diff
@@ -34,5 +34,29 @@ public PokemonExperience(int currentLevel, int baseExperience, int totalLevelExp
             }
             TotalLevelExperience = totalLevelExperience;
         }
+
+        // A value between 0 and 54 that represents how much exp the Pokemon has gained since its last level.
+        public int RatioToNextLevel

```

Comment:
Why between 0 and 54? This seems like an arbitrary choice.
A float between 0 and 1 or between 0 and 100 it probably more generic.

Reply:
I just copy-pasted the way PRO does it.

Reply:
If you or anyone else can figure out how to get the "experience gained since last level" I would gladly change this